src/AutomationFramework/Config.java

```diff
@@ -0,0 +1,85 @@
+package AutomationFramework;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+
+public class Config {
+	public String FireFoxExePath = "C:\\Downloaded Things\\geckodriver.exe";
+	public String ChromeExePath = "C:\\Downloaded Things\\chromedriver.exe";
+	//public String WebsiteURL = "https://california.demo.collaborativefusion.com/";
+	public String WebsiteURL = "https://www.pulsedemo.org";
+	public String ValidCredFilePath = "C:\\Julie\\6-PULSE\\PULSECredentials.txt";
+	public String PatientSearchFile = "C:\\Julie\\6-PULSE\\SearchInput.txt";
+	
+	@Test
+	public WebDriver Open_FireFox() throws InterruptedException {
+				System.setProperty("webdriver.gecko.driver", FireFoxExePath);
+				WebDriver driver = new FirefoxDriver();
+				driver.get(WebsiteURL);
+
+				System.out.println("Successfully opened PULSE website");
```

Comment:
Selenium tests out to use something from the "ort.testng.Assert" library to assert tests, and most likely only use System.out.println for debugging, but almost never have those lines committed